modules/account-lib/src/coin/dot/utils.ts

```diff
@@ -0,0 +1,156 @@
+import * as hex from '@stablelib/hex';
+import base32 from 'hi-base32';
+import { Keyring, decodeAddress, encodeAddress } from '@polkadot/keyring';
+import { hexToU8a, isHex } from '@polkadot/util';
+import { base64Decode } from '@polkadot/util-crypto';
+import { decodePair } from '@polkadot/keyring/pair/decode';
+import { KeyPair } from '.';
+import { BaseUtils } from '../baseCoin';
+import { NotImplementedError } from '../baseCoin/errors';
+import { Seed } from './iface';
+import { TypeRegistry } from '@substrate/txwrapper-core/lib/types';
+import { EXTRINSIC_VERSION } from '@polkadot/types/extrinsic/v4/Extrinsic';
+import { createMetadata, construct, getRegistry } from '@substrate/txwrapper-polkadot';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { UnsignedTransaction } from '@substrate/txwrapper-core';
+import { testnetMetadataRpc } from './metadataRpc';
+const polkaUtils = require('@polkadot/util');
+const { createTypeUnsafe } = require('@polkadot/types');
+
+export class Utils implements BaseUtils {
+  /** @inheritdoc */
+  isValidAddress(address: string): boolean {
```

Comment:
https://wiki.polkadot.network/docs/build-transaction-construction -> Derive address

Reply:
I dont understand what you mean here

Reply:
Checked with Jorge derive address not be used here.